tests/transforms/test_metric_tensor.py

```diff
@@ -1742,6 +1742,25 @@ def circuit_multi_block(x, z):
     assert len(recwarn) == 0
 
 
+def test_raises_circuit_that_uses_missing_wire():
+    """Test that an error in the original circuit is reraised properly
+    and not caught, leading to a MaxRecursionError."""
+

```

Comment:
not a `qml.wire.WireError` ?

Reply:
Ah, I phrased this ambiguously. I meant that the catching _would_ lead to a recursion error :sweat_smile: Will rephrase.